README.md

```diff
@@ -1,2 +1,259 @@
+# PagoPA eCommerce user stats service
+
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=pagopa_pagopa-ecommerce-user-stats-service&metric=alert_status)](https://sonarcloud.io/dashboard?id=pagopa_pagopa-ecommerce-user-stats-service)
+
+This microservice is responsible for expose stats for user performing transactions on eCommerce.
+
+Those stats include information about the latest used payment method for a transaction
+
+- [PagoPA eCommerce user stats service](#pagopa-ecommerce-user-stats-service)
+    * [Api Documentation 📖](#api-documentation-)
+    * [Technology Stack](#technology-stack)
+    * [Start Project Locally 🚀](#start-project-locally-)
+        + [Prerequisites](#prerequisites)
+        + [Run docker container](#run-docker-container)
+    * [Develop Locally 💻](#develop-locally-)
+        + [Prerequisites](#prerequisites-1)
+        + [Run the project](#run-the-project)
+          + [Install ecommerce commons library](#install-ecommerce-commons-library-locally)
+        + [Testing 🧪](#testing-)
+            - [Unit testing](#unit-testing)
+            - [Integration testing](#integration-testing)
+            - [Performance testing](#performance-testing)
+    * [Dependency management 🔧](#dependency-management-)
+        + [Dependency lock](#dependency-lock)
+        + [Dependency verification](#dependency-verification)
+    * [Contributors 👥](#contributors-)
+        + [Maintainers](#maintainers)
+
+
+---
+
+## Api Documentation 📖
+
+See
+the [OpenAPI 3 here.](https://editor.swagger.io/?url=TODO)
+
+---
+
+## Technology Stack
+
+- Kotlin
+- Spring Boot
+
+---
+
+## Start Project Locally 🚀
+
+### Prerequisites
+
+- docker
+
+### Populate the environment
+
+The microservice needs a valid `.env` file in order to be run.
+
+If you want to start the application without too much hassle, you can just copy `.env.example` with
+
+```shell
+$ cp .env.example .env
+```
+
+to get a good default configuration.
+
+If you want to customize the application environment, reference this table:
+
+| Variable name      | Description                                                  | type   | default |
+|--------------------|--------------------------------------------------------------|--------|---------|
+| ROOT_LOGGING_LEVEL | Root logging level                                           | string | INFO    |
+| APP_LOGGING_LEVEL  | Application logging level (package it.pagopa)                | string | INFO    |
+| WEB_LOGGING_LEVEL  | Spring web logging level (logs about http requests/responses | string | OFF     |
+
+### Run docker container
+
+```shell
+$ docker compose up --build
+```
+
+---
+
+## Develop Locally 💻
+
+### Prerequisites
+
+- git
+- gradle
+- jdk-17
+
+### Run the project
+
+```shell
+$ export $(grep -v '^#' .env.local | xargs)
+$ ./gradlew bootRun
+```
+
+### Install eCommerce commons library locally
```

Comment:
remove this section

Reply:
fixed with https://github.com/pagopa/pagopa-ecommerce-user-stats-service/pull/1/commits/302f276de3f8eecff1fe3504e50e54c7bdc4546c